.circleci/config.yml

```diff
@@ -133,9 +141,37 @@ jobs:
       - run: yarn install
       - run: ./scripts/promote
 
+  run-all-nuts:
+    docker:
+      - image: node:latest
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v3-npm-{{checksum "yarn.lock"}}
+      - run: yarn install
+      - run: ./scripts/run-plugin-nuts.js
+
+  monitor-all-nuts:
+    docker:
+      - image: node:latest
+    parameters:
+      nuts_job_data:
+        description: 'Parameter to pass the set of NUTs started across all plugins'
+        default: '[]'
+        type: string
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v3-npm-{{checksum "yarn.lock"}}
+      - run: yarn install
+      - run: ./scripts/monitor-plugin-nuts.js <<parameters.nuts_job_data>>
+
 workflows:
   version: 2.1
   test-and-release:
+    when: << pipeline.parameters.run-auto-workflows >>
```

Comment:
Condition that controls if a workflow will run. For typical PR/Merge activities, this workflow is triggered by GH events. The default value for `run-auto-workflows` is true, so this workflow will run.